README.md

```diff
@@ -1,135 +1,61 @@
-# Auction system
+[![Lagom](docs/logo.png)](https://github.com/lagom/lagom)
 
-System architecture:
+[![Gitter](https://img.shields.io/gitter/room/gitterHQ/gitter.svg)](https://gitter.im/lagom/lagom?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)[<img src="https://img.shields.io/travis/lagom/lagom.svg"/>](https://travis-ci.org/lagom/lagom)
 
-# Item service
+# INTRO
+Lagom is a Swedish word meaning just right, sufficient. Microservices are about creating services that are just the right size, that is, they have just the right level of functionality and isolation to be able to adequately implement a scalable and resilient system.
 
-Manages the description and auction status (created, auction, completed, cancelled) of an item.
+Lagom focuses on ensuring that your application realises the full potential of the [Reactive Manifesto](http://reactivemanifesto.org/), while delivering a high productivity development environment, and seamless production deployment experience.
 
-## Queries handled
+This is a sample Scala auction site using the Lagom Framework. A java version of the auction site is also available [here](https://github.com/lagom/online-auction-java) 
```

Comment:
"There's also a [java version](https://github.com/lagom/online-auction-java) of this sample app available."

I prefer not using the word 'here' for linking.